R/build-articles.R

```diff
@@ -68,21 +76,31 @@ build_articles <- function(pkg = ".", path = "docs/articles", depth = 1L,
   rule("Building articles")
   mkdir(path)
 
+  vignette_path <- file.path(pkg$path, "vignettes")
```

Comment:
I don't think the times should be computed here, but should instead be computed in `build_article()`